src/js/components/core/login/GogsApi.js

```diff
@@ -3,130 +3,123 @@
  * @author - Ian Hoegen
  ******************************************************************/
 const Gogs = require('gogs-client');
-const api = new Gogs('https://git.door43.org/api/v1'), tokenStub = {name: 'translation-core'};
+const api = new Gogs('https://git.door43.org/api/v1'), tokenStub = { name: 'translation-core' };
 var CryptoJS = require("crypto-js");
 
-function UserManager(auth) {
-  return {
 /**
   * @description - Login a user and get a user object with token back.
   * @param {Object} userObj - Must contain fields username and password
   * @return {Promise} - Returns a promise with a user object.
   *****************************************************************/
-    login: function (userObj) {
-      return api.getUser(userObj).then(function (user) {
-        return api.listTokens(userObj)
-        .then(function (tokens) {
-          return tokens.find((el) => el.name == tokenStub.name);
-        })
-        .then(function (token) {
-          return token ? token : api.createToken(tokenStub, userObj);
-        })
-        .then(function (token) {
-          var phrase = window.ModuleApi.getAuthToken('phrase') != undefined ? window.ModuleApi.getAuthToken('phrase') : "tc-core";
-          var encryptedToken = CryptoJS.AES.encrypt(JSON.stringify(userObj), phrase);
-          localStorage.setItem('user', encryptedToken);
-          user.token = token.sha1;
-          return user;
-        });
+export function login(userObj) {
+  return api.getUser(userObj).then(function (user) {
+    return api.listTokens(userObj)
+      .then(function (tokens) {
+        return tokens.find((el) => el.name == tokenStub.name);
+      })
+      .then(function (token) {
+        return token ? token : api.createToken(tokenStub, userObj);
+      })
+      .then(function (token) {
+        var phrase = window.ModuleApi.getAuthToken('phrase') != undefined ? window.ModuleApi.getAuthToken('phrase') : "tc-core";
+        var encryptedToken = CryptoJS.AES.encrypt(JSON.stringify(userObj), phrase);
+        localStorage.setItem('user', encryptedToken);
+        user.token = token.sha1;
+        return user;
       });
-    },
+  });
+}
 /**
   * @description - Create an account for a user.
   * @param {Object} user - Must contain fields username, password, and email.
   * @return {Promise} - Returns a promise with a user object.
   *****************************************************************/
-    createAccount: function (user) {
-      return api.createUser(user, auth, true)
-      .then(function(updatedUser) {
-        return api.createToken(tokenStub, user)
-        .then(function(token) {
+export function createAccount(user) {
+  return api.createUser(user, auth, true)
```

Comment:
`auth` will not be defined because it is passed down by the `userManager` function.